app/parsers/bulkrax/csv_parser.rb

```diff
@@ -39,10 +39,11 @@ def required_elements
       %w[title source_identifier]
     end
 
-    def validate_import
-      raise "Missing required elements, required elements are: #{required_elements.join(', ')}" unless required_elements?(import_fields)
-      # check file_paths; this will raise an error if any files are missing
-      true if file_paths.present?
+    def valid_import?
```

Comment:
see comment, changing this to always return true or false and handling the error otherwise we end up with pointless retries.